src/test/java/org/fundacionjala/coding/denis/TestFizzBuzz.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.denis;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This is the class test principal.
+ */
+public class TestFizzBuzz {
```

Comment:
Your implementation is failing for the following scenario.
        assertEquals("Buzz", data.verifyFizzBuzz(35));